### PR TITLE
Update Air Gap Instructions to Use Scan 2.0 with Trivy

### PR DESCRIPTION
**Pull request by ryan-a-baker**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?

None, this is a 1.8 capability.

It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

**Comment by ryan-a-baker** [comment]
This PR needs https://github.com/pivotal/docs-tap/pull/3251 merged to included the external DB updates before this is merged.

File: partials/scst-scan/_offline-airgap.hbs.md [hunk 1]
```diff
@@ -1,3 +1,69 @@
+## <a id="trivy-with-scan-2-0"></a> Using Trivy with Scan 2.0
+
+The Aqua Trivy vulnerability scanner uses two databases to perform vulnerability scans:
+
+1. Vulnerability Database: This database contains the vulnerability information that is used to scan artifacts. This database is built every six hours on [GitHub](https://github.com/aquasecurity/trivy-db) and is distributed via [GitHub Container registry (GHCR)](https://ghcr.io/aquasecurity/trivy-db). 
+
+2. Java Index Database: This database enables Trivy to identify the groupId, artifactId, and version of JAR files. It is built once a day on [GitHub](https://github.com/aquasecurity/trivy-java-db) and distributed via [GitHub Container registry (GHCR)](https://ghcr.io/aquasecurity/trivy-java-db). 
+
+In an online installation of Tanzu Application Platform, these databases are automatically downloaded from GHCR with each scan execution.  In order to enable scanning in an offline environment, these two databases need to be relocated to your private registry, and Trivy needs to be configured to use your private location instead of downloading from GHCR.
```
Comment (rxl7906) [inline partials/scst-scan/_offline-airgap.hbs.md:9]: Nit: I'm assuming you got some of these lines from the aquasecurity docs here https://aquasecurity.github.io/trivy/v0.49/docs/scanner/vulnerability/? Should we add a section, for more info check out these docs for? But I'm okay if we don't add it.

File: partials/scst-scan/_offline-airgap.hbs.md [hunk 2]
```diff
@@ -1,3 +1,69 @@
...
+
```
Comment (rxl7906) [inline partials/scst-scan/_offline-airgap.hbs.md:10]: Nit: Trivy also provides a Air-Gapped Environment section [here](https://aquasecurity.github.io/trivy/v0.49/docs/advanced/air-gap/) where it details the vulnerability database download, trivy with specify flags in air-gapped environment, and configurations. Should we link this page here and say "for more details about Air-Gapped see aquasec docs"?

File: partials/scst-scan/_offline-airgap.hbs.md [hunk 3]
```diff
@@ -1,3 +1,69 @@
...
+### <a id="relocate-trivy-databases"></a>Relocate Trivy Images
+
+The databases are stored in an OCI compliant registry as OCI artifacts.  In order to copy these to your private registry, Trivy recommends using the oras cli.  It can be installed using the following instructions.
```
Comment (rxl7906) [inline partials/scst-scan/_offline-airgap.hbs.md:13]: > Trivy recommends using the oras cli

Should we link the oras cli here? https://oras.land/docs/installation/

**Review by rxl7906 — APPROVED** [review]
LGTM, the comments were suggestions to add links for more context but not necessary now because this is a first pass on airgap scan 2.0 trivy.

File: partials/scst-scan/_offline-airgap.hbs.md (hunk 3 above)
Comment (ryan-a-baker) [reply partials/scst-scan/_offline-airgap.hbs.md:13]: Good call.  I meant to make that a link but clearly didn't :)

**Comment by ryan-a-baker** [comment]
https://github.com/pivotal/docs-tap/pull/3309

**Pull request merged**